src/util/analytics-client.ts

```diff
@@ -10,12 +10,12 @@ const logger = getLogger("analytics");
 let analyticsNodeClient;
 
 export const sendAnalytics: {
-	(eventType: "screen", attributes: { name: string } & Record<string, unknown>);
-	(eventType: "track", attributes: { name: string, source: string } & Record<string, unknown>);
-	(eventType: "ui" | "operational", attributes: Record<string, unknown>);
-} = (eventType: string, attributes: Record<string, unknown> = {}): void => {
+	(jiraHost: string, eventType: "screen", attributes: { name: string } & Record<string, unknown>);
+	(jiraHost: string, eventType: "track", attributes: { name: string, source: string } & Record<string, unknown>);
+	(jiraHost: string, eventType: "ui" | "operational", attributes: Record<string, unknown>);
+} = (jiraHost: string, eventType: string, attributes: Record<string, unknown> = {}): void => {
 
-	logger.debug(analyticsClient ? "Found analytics client." : `No analytics client found.`);
+	logger.info({ jiraHost }, analyticsClient ? "Found analytics client." : `No analytics client found.`);
```

Comment:
/nit 
Are you sure to turn this to info? As there will be a lot of logs?

Reply:
That's exactly what I want to check, how much events do we send per jiraHost :) I'll remove it once we have this data